app.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Botpress",
+  "description": "The Conversational Platform with built-in language understanding (NLU), beautiful graphical interface and Dialog Manager (DM).",
+  "repository": "https://github.com/botpress/botpress"
```

Comment:
Heroku supports having a custom logo in the app creation page when the button is clicked. If we had a logo stored somewhere this would be a nice +
`"logo": "https://botpress.com/logo.png",`